incubator/aci/create-k8-connector/script.md

```diff
@@ -114,30 +121,10 @@ spec:
 kubectl create -f examples/nginx-pod.yaml 
 ```
 
-```
-kubectl get pods -o wide
-```
-
-Since we need to ensure our Public IPs have been assigned before
-proceeding, and because we need the IP number later we'll run a loop
-to grab the IP once assinged. This is a little cumbersome but great if
-you want to script things. If you are doing this manually you can use
-`kubectl get service --wait` to display changes as they happen.
-
-```
-NGINX_IP=""
-while [ -z $NGINX_IP ]; do sleep 10; NGINX_IP=$(kubectl get service vamp -o jsonpath="{.status.loadBalancer.ingress[*].ip}"); done
```

Comment:
Why are we removing this? Your comment on the PR says it's necessary to remove it but not why. There is no replacement for this logic and thus we are left without demonstrating that the container is actually running.

Reply:
I tested the line in bash and it didn't work as expected

Reply:
So you don't want to wait for an IP? Isn't it better to fix it rather than remove it entirely? The problem is that you are looking up the wrong service name (it's not 'vamp')